core/src/test/java/org/apache/brooklyn/util/core/logbook/file/FileLogStoreTest.java

```diff
@@ -48,64 +49,74 @@
             "\tat org.apache.felix.scr.impl.ComponentRegistry.checkComponentName(ComponentRegistry.java:240) ~[?:?]\n" +
             "\tat org.apache.felix.scr.impl.BundleComponentActivator.validateAndRegister(BundleComponentActivator.java:443) ~[?:?]";
 
+    private final AtomicInteger lineCount = new AtomicInteger();
+
     @BeforeTest
     public void setUp() {
-        TimeZone.setDefault(TimeZone.getTimeZone("GMT"));
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
i don't like having to set this in a test but don't have a better idea so go for it